tools/create-esm.js

```diff
@@ -0,0 +1,9 @@
+const fs = require('fs')
+
+const exportRegex = /module.exports\s*=\s*pathToRegexp/
+
+const cjsSource = fs.readFileSync('index.js', { encoding: 'utf8' })
+
+const esmSource = cjsSource.replace(exportRegex, 'export default pathToRegexp\nexport { parse, compile, tokensToFunction, tokensToRegExp }\n')
```

Comment:
Unfortunately I won't be able to accept this line as is. It actually deviates from existing usages meaning this is a breaking change (e.g. anyone using TypeScript or Webpack could be doing `import * as pathToRegexp` and using it as a function).